<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Server Control Panel</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Лог</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QTextEdit" name="logBox">
           <property name="maximumSize">
            <size>
             <width>2000</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 255, 0);</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Контрольная панель</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="sizeConstraint">
              <enum>QLayout::SetMinAndMaxSize</enum>
             </property>
             <item>
              <widget class="QPushButton" name="clearResourcesButton">
               <property name="text">
                <string>Освободить
ресурсы</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="rejectAuthButton">
               <property name="text">
                <string>Запретить
регистрацию</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignVCenter">
              <widget class="QPushButton" name="rejectRcButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Отклонять новые
запросы на ресурсы</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Время доступа к ресурсу (c):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="accessTimeSpinBox">
               <property name="maximumSize">
                <size>
                 <width>97</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="specialValueText">
                <string/>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>3600</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="acceptAccessTime">
               <property name="maximumSize">
                <size>
                 <width>97</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Применить</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Порт:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="portSpinBox">
               <property name="maximumSize">
                <size>
                 <width>97</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="maximum">
                <number>65535</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="switchPortButton">
               <property name="maximumSize">
                <size>
                 <width>97</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Открыть порт</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Информационная панель</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="sizeConstraint">
                <enum>QLayout::SetDefaultConstraint</enum>
               </property>
               <item>
                <widget class="QLabel" name="label">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Прошло времени со старта:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="uptime">
                 <property name="maximumSize">
                  <size>
                   <width>108</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>00:00:00</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Разрешенное количество пользователей:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="userCount">
                 <property name="maximumSize">
                  <size>
                   <width>108</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>150</height>
                </size>
               </property>
               <property name="title">
                <string>Список ресурсов</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_10">
                  <item>
                   <widget class="QLabel" name="label_11">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>67</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Ресурс №1:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="resource1">
                    <property name="text">
                     <string>Ресурс свободен</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_11">
                  <item>
                   <widget class="QLabel" name="label_13">
                    <property name="maximumSize">
                     <size>
                      <width>67</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Ресурс №2:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="resource2">
                    <property name="text">
                     <string>Ресурс свободен</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_12">
                  <item>
                   <widget class="QLabel" name="label_15">
                    <property name="maximumSize">
                     <size>
                      <width>67</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Ресурс №3:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="resource3">
                    <property name="text">
                     <string>Ресурс свободен</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_13">
                  <item>
                   <widget class="QLabel" name="label_17">
                    <property name="maximumSize">
                     <size>
                      <width>67</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Ресурс №4:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="resource4">
                    <property name="text">
                     <string>Ресурс свободен</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>300</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>339</height>
                </size>
               </property>
               <property name="title">
                <string>Подключенные пользователи</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QListWidget" name="connectedClientsList"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="maximumSize">
         <size>
          <width>145</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Текущий файл настроек:</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="settingsPathLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>314</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>218</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(175, 175, 175);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reloadSettings">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>122</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Перезагрузить файл
настроек</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
